Procfile.dev

```diff
@@ -1,4 +1,3 @@
 web: ./bin/rails server -p 3000 -b 0.0.0.0
 js: yarn build:js --watch
 css: yarn build:css --watch
-worker: ./bin/rails jobs:work
```

Comment:
> In Rails' development environment, by default, GoodJob's Adapter executes jobs async in a background thread pool in rails server

https://github.com/bensheldon/good_job?tab=readme-ov-file#set-up

I assume this is what we want for development right? Does it make sense to have it running as a standalone process? 🤔

Reply:
could be, I guess we need to choose between ActiveJob or GoodJob inline adapters?

Reply:
Yeah, I think using GoodJob's default setup is ok

Reply:
I just pushed a commit using good_job in all envs, but now I realize I didn't get your comment. GoodJob don't need an external process for dev env because it uses a background thread within the rails server, right?

>In Rails' development environment, by default, GoodJob's Adapter executes jobs async in a background thread pool in rails server.

Reply:
Yes, that's correct! We could still use the external process for dev if we want to, but I think the background thread is ok for development. My comment was just to confirm that 🙂